src/index.ts

```diff
@@ -33,13 +33,24 @@ export {
   setExtras,
   setTag,
   setTags,
+  flush,
   setUser,
   // eslint-disable-next-line deprecation/deprecation
   startTransaction,
+  withScope,
+
+  // V8
+  startInactiveSpan,
   startSpan,
   startSpanManual,
-  startInactiveSpan,
-  withScope,
+  getActiveSpan,
+  spanToJSON,
+  spanIsSampled,
+  setMeasurement,
+  getGlobalScope,
```

Comment:
I've noticed current scope missing.

```suggestion
  getGlobalScope,
  getCurrentScope,
```

Reply:
Ohh, I see it's already on line 26.